OrbitGl/App.cpp

```diff
@@ -344,6 +346,25 @@ void OrbitApp::AddSymbol(uint64_t a_Address, const std::string& a_Module,
   symbol->m_Module = a_Module;
   Capture::GTargetProcess->AddSymbol(a_Address, symbol);
 }
+//-----------------------------------------------------------------------------
+void OrbitApp::AddKeyAndString(uint64_t key, const std::string& str) {
+  if (ConnectionManager::Get().IsService()) {
```

Comment:
after PR#239 goes in OrbitApp is no longer constructed for the service.  In mentioned PR the code moved to ConnectionManager

Reply:
So, where should this method move, also ConnectionManager?

Reply:
Yes, but I will amend 239 accordingly if this PR goes in first.

Reply:
Ok, thanks!